modules/native/feathers/math_extra/math_extra_ext.cpp

```diff
@@ -247,7 +247,7 @@ namespace Falcon {
             Item *num2 = vm->param( 1 );
 
             // The arguments must be ordinal and must be greater than 0
-            if ( num1 <= 0 || ! num1->isOrdinal() || num2 <= 0 || ! num2->isOrdinal() )
+            if ( num1 <= (Item*) 0 || ! num1->isOrdinal() || num2 <= (Item*) 0 || ! num2->isOrdinal() )
```

Comment:
The 0 here was meant to be an ordinal; there was an automatic conversion to/from number in comparisons. Here we need to use the correct asInteger/asNumber.

Reply:
Try with (*num1) <= 0   it should work.